### DISCUSSION TITLE
Rilevare lo stato di un contatto pulito e ripeterlo

### DISCUSSION THREAD

**Discussion by Febbocop** [indomus/forum#2812] [Hardware & Firmware]
Buongiorno ragazzi, sono nuovo e sono un appassionato di domotica, mi trovo a dover risolvere un problema, ma non so come fare.
A causa di un vecchio impianto di riscaldamento, cambiato e adattato negli anni, mi ritrovo ad avere due 4 termostati in casa su due zone, in pratica ho due termostati per zona, uno controlla l'accensione e lo spegnimento della caldaia e il secondo apre e chiude la valvola della relativa zona e mi ritrovo a dover fare i salti mortali per far combaciare i due termostati della relativa zona.
Vorrei ridurli a due in modo che all'accensione della caldaia ordinata dal termostato di una delle due zone corrisponda anche l'apertura della valvola  di quella zona appunto.
Stavo quindi cercando una soluzione domotica a questo problema per cui vi chiedo se esiste una switch a contatto pulito che possa rilevare, collegato in parallelo con l'attuatore che controlla l'accensione e lo spegnimento della caldaia, l'apertura o la chiusura del circuito e ripetere la stessa azione su un secondo switch che andrebbe ad aprire o chiudere la valvola  di una delle due zone.
In poche parole dovrei rilevare lo stato di un contatto pulito e riportarlo, anche tramite una routine, su un altro switch\attuatore.
Vi ringrazio in anticipo e spero di essermi riuscito a spiegare.

**Comment by KC-inDomus** [comment]
Beh, tutto questo lo fai facilmente tramite automazioni Home Assistant o altri HUB personali di sorta. Ti bastano 4 attuatori a contatto pulito, integrati sull'HUB, due li usi come termostati virtuali e due per attuare le valvole.

**Reply by Febbocop** [reply]
E lo so, ma era per evitare di eliminare i due cronotermostati che avevo messo da poco, che si programmano molto facilmente e velocemente, anche "al volo" senza bisogno di app o pc.
Questi cronotermostati hanno un attuatore rf che agisce sulla caldaia, quindi chiedevo se esiste qualcosa di collegabile in parallelo che possa rilevare lo stato del suddetto attuatore e poi di conseguenza agire su un altro attuatore messo sulla valvola di zona.
In qualsiasi caso, grazie mille della risposta.

**Comment by 50494554524F** [comment]
vedi un po' se può fare al caso tuo

https://github.com/pvvx/ATC_MiThermometer#reed-switch-or-button-functions

**Comment by Febbocop** [comment]
Ringrazio chi ha provato a darmi una mano, alla fine ho risolto con un Sonoff S-Mate 2 che mi legge lo stato del contatto pulito e un sonoff mini r4 collegato a un rele a contatto pulito che mi apre le valvole di zona.
